packages/frontend/core/src/blocksuite/ai/chat-panel/chat-context.ts

```diff
@@ -1,40 +1,12 @@
+import type {
+  ChatStatus,
+  HistoryMessage,
+} from '../components/ai-chat-messages';
 import type { AIError } from '../provider';
 
-export type ChatMessage = {
-  id: string;
-  content: string;
-  role: 'user' | 'assistant';
-  attachments?: string[];
-  createdAt: string;
-};
-
-export type ChatAction = {
-  action: string;
-  messages: ChatMessage[];
-  sessionId: string;
-  createdAt: string;
-};
-
-export type ChatItem = ChatMessage | ChatAction;
-
-export function isChatAction(item: ChatItem): item is ChatAction {
-  return 'action' in item;
-}
-
-export function isChatMessage(item: ChatItem): item is ChatMessage {
-  return 'role' in item;
-}
-
-export type ChatStatus =
-  | 'loading'
-  | 'success'
-  | 'error'
-  | 'idle'
-  | 'transmitting';
-
 export type ChatContextValue = {
   // history messages of the chat
-  items: ChatItem[];
+  messages: HistoryMessage[];
```

Comment:
Maybe we could split HistoryMessage to UserMessage/AssistantMessage/ActionMessage/... future

Reply:
I will separate the chat context later, but not in this PR.